packages/util/src/types.ts

```diff
@@ -118,3 +118,20 @@ export function toType<T extends TypeOutput>(
       throw new Error('unknown outputType')
   }
 }
+
+type Falsy = false | '' | 0 | null | undefined
+
+export function isFalsy(value: unknown): value is Falsy {
```

Comment:
Can you add a very very explicit note here (and for `isTruthy` as well) that this is a function which should only temporarily be used and otherwise be replaced with the direct checks (as you mentioned as well), together with some overall context why this function exists?

I otherwise have the strong fear that this looks like as something "how we do things" and other developers rather align with this.

Reply:
Good point. I added a note and marked them as deprecated. The goal should definitely to get rid of those eventually, and it should not be used in new code (I think the `@deprecated` natspec will help achieve that)

Reply:
Thanks, that's a good solution! 👍 😄